Andrii_Voinenko/Lesson14/parent.py

```diff
@@ -0,0 +1,25 @@
+from abc import ABC, abstractmethod
+
+
+class Parent(ABC):
+    def __init__(self, name: str, gender: str, age: int):
+        # Hiding
+        self._name = name
+        self._gender = gender
+        self._age = age
+
+        if self._age < 18:
+            raise TypeError('Available only for 18+')
```

Comment:
TypeError | Raised when a function or operation is applied to an object of the incorrect type(int, str, etc)
Maybe, ValueError more correct than TypeError 



<!--EndFragment-->
</body>
</html>